README.md

```diff
@@ -1,20 +1,75 @@
+<h1 align="center">
+    <img margin="0 10px 0 0" src="https://www.docker.com/wp-content/uploads/2022/03/vertical-logo-monochromatic.png" width="150px"/>
+    <img margin="0 10px 0 0" src="https://uploads-ssl.webflow.com/60d1a7f5aeb33cb8af546898/610bcf4cc7ae73979fa0d23b_256.png" width="120px"/>
+</h1>
+  <h2 align="center">Picsellia Public Repo</h2>
+  <p align="center">A collection of Docker images provided for your trainings!
+<p>
+<p align="center">
+    <a href="https://www.python.org/downloads/" target="_blank"><img src="https://img.shields.io/badge/python-3.8%20|%203.9%20|%203.10-brightgreen.svg" alt="Python supported"/></a>
+</p>
 
+The Picsellia Training Engine is a project designed to provide an efficient, customizable, and Docker-based system for
+training ML models. Its main purpose is to interact seamlessly with the Picsellia SAS platform,
```

Comment:
I think it is useful to present what the training scripts are for. something along the lines of: 
"These images have a twofold purpose: launching the training and evaluations for CV tasks (image classification, object detection and image segmentation) using a selection of SOTA models, while ensuring a seamless interaction with the Picsellia SAS platform.